pymc/distributions/continuous.py

```diff
@@ -2985,7 +2985,9 @@ def triangular_default_transform(op, rv):
 
 class Gumbel(Continuous):
     r"""
-    Univariate Gumbel log-likelihood.
+    Univariate Gumbel right Skew log-likelihood. This parameterization will provide the extrem maximum value. 
+    Those looking to find the extrem minimum provided by the left skew Gumbel should 
+    invert the sign to all X and Y values (positive to negative and negative to positive).
```

Comment:
```suggestion
    Univariate Gumbel right Skew log-likelihood. This parameterization will provide the extreme maximum value.
    Those looking to find the extreme minimum provided by the left skewed Gumbel should
    invert the sign to all X and Y values (positive to negative and negative to positive).
```
fixes two typos and removes trailing whitespace which our `pre-commit` would complain about

Reply:
It also needs to be formatted as 1st sentence, a white line and then the more extense description to follow numpydoc: https://numpydoc.readthedocs.io/en/latest/format.html#sections

Reply:
Hell, sorry for not looking at these comments. I am dealing with wedding planning and work at the same time. will look into this around the end of February.